use expressions as first choice in all docs exmples (#10118)
circleci/circleci-docs: docs/guides/modules/orchestrate/pages/workflows.adoc

    triggers:
      - schedule:
          cron: "0 0 * * *"

>           filters: pipeline.git.branch == "main" or pipeline.git.branch matches /^release\/.*/ # only run the nightly workflow on the main branch and any branch that starts with `release/`

Reviewer: I am not certain that we changed config processing for the scheduled workflows case (since we want scheduled workflows to go away). Will need to check up on this one.

Author: I'm struggling to verify this one - even using old style filters I'm having issues actually 👎

  build: # This workflow will run on all branches except 'main' and will not run on tags
    jobs:
      - test:

>           filters: pipeline.git.branch != main # only run the workflow on any branch that is not the main branch

Reviewer 2: YAML anchor syntax is invalid here. `value &anchor` puts `&filters-staging` inside the scalar string — it does not create a YAML anchor. Anchors must precede the node they name.

```suggestion
          filters: &filters-staging pipeline.git.branch == "main" # this yaml anchor is setting these values to "filters-staging"
```

Same issue on the `&filters-production` line below.

>           filters: pipeline.git.branch != main # only run the workflow on any branch that is not the main branch

Reviewer 2: Unquoted string. `main` is not a keyword — every other comparison in this PR uses quoted strings. Should be `"main"`.

```suggestion
          filters: pipeline.git.branch != "main" # only run the workflow on any branch that is not the main branch
```

  production: # This workflow will only run on tags (specifically starting with 'v.') and will not run on branches
    jobs:
      - test:

>           filters: pipeline.git.tag matches /^v.*/ &filters-production # this yaml anchor is setting these values to "filters-production"

Reviewer 2: Same YAML anchor problem — `&filters-production` must precede the scalar value:

```suggestion
          filters: &filters-production pipeline.git.tag matches /^v.*/ # this yaml anchor is setting these values to "filters-production"
```
